nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -862,7 +862,8 @@ in {
               `pkgs.nextcloud`.
             ''
           else if versionOlder stateVersion "24.05" then nextcloud27
-          else nextcloud29
+          else if versionOlder stateVersion "24.11" then nextcloud28
+          else nextcloud30
```

Comment:
I think we don't need the `else if` for `24.11` here, just do `else nextcloud30`: if you have a fresh NixOS install (i.e. stateVersion = 24.11) you ended up on nextcloud29 originally, so going straight to nextcloud30 is still safe since we move forward one major-release only.

Or am I missing something?

Reply:
I think anyway the `netcloud28` would be wrong as  it would mean a downgrade for current 24.11 users.

But I agree with @Ma27 that we should just go to `nextcloud30` and no if needed

Reply:
Maybe we can make things simpler going forward if we don't specify a default package at all.

Reply:
I would like to keep the default package version, it will help automatically upgrade major Nextcloud version on new state versions